.github/scripts/context_chunker.py

```
@@ -0,0 +1,87 @@
+        # Setup tokenizer/encoder if tiktoken available
+        self._encoder = None
+        if TIKTOKEN_AVAILABLE:
+            try:
+                # Use a common 32k context model encoding if available; fallback to cl100k_base
+                self._encoder = tiktoken.get_encoding("cl100k_base")
+            except Exception as e:
+                print(f"Warning: failed to initialize tiktoken encoder: {e}", file=sys.stderr)
+                self._encoder = None
+
+        # Precompiled regexes or any other helpers
+        processed_content = self._build_limited_content(chunks)
```

Ruff (0.14.6)

Undefined name `chunks`

(F821)